domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementsController.cs

```diff
@@ -42,20 +46,30 @@ public async Task<ActionResult> Create([FromBody] CreateTransferAgreement reques
 
         var result = await transferAgreementRepository.AddTransferAgreementToDb(transferAgreement);
 
-        var response = new TransferAgreementDto(
-           Id: result.Id,
-           StartDate: result.StartDate.ToUnixTimeSeconds(),
-           EndDate: result.EndDate.ToUnixTimeSeconds(),
-           SenderName: result.SenderName,
-           SenderTin: result.SenderTin,
-           ReceiverTin: result.ReceiverTin);
+        return CreatedAtAction(nameof(Get), new { id = result.Id }, ToTransferAgreementDto(result));
+    }
+
+    [ProducesResponseType(typeof(TransferAgreement), 200)]
+    [ProducesResponseType(404)]
```

Comment:
If you change this to 

```suggestion
    [ProducesResponseType(typeof(void), 404)]
```

then the open api definition will say that there is no content returned. The open api definition is currently saying that it returns `ProblemDetails`